tools/envoy_headersplit/changed.txt

```diff
@@ -0,0 +1,147 @@
+test/common/access_log/access_log_impl_test.cc
```

Comment:
Do we need to check this in?

Reply:
I will move it to the corresponding unit test